src/main/scala/temple/generate/server/go/service/main/GoServiceMainCreateHandlerGenerator.scala

```diff
@@ -42,7 +42,14 @@ object GoServiceMainCreateHandlerGenerator {
     )
   }
 
-  private def generateDAOCallBlock(root: ServiceRoot, usesMetrics: Boolean, metricSuffix: Option[String]): String =
+  private def generateDAOCallBlock(root: ServiceRoot, usesMetrics: Boolean, metricSuffix: Option[String]): String = {
```

Comment:
Could split this function in two, to have `generateDAOCallBlock` and `generateDAOCallErrorBlock` to match the other handler generators. Though I'm not too fussed